src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -0,0 +1,352 @@
+
+```python
+import sentry_sdk
+    
+sentry_sdk.init("___PUBLIC_DSN___", traces_sample_rate=0.1)
+```
+
+**Automatic Instrumentation**
+
+Many integrations for popular frameworks automatically capture traces. If you already have any of the following frameworks set up for Sentry error reporting, you will start to see traces immediately:
+
+- All WSGI-based frameworks (Django, Flask, Pyramid, Falcon, Bottle)
+- Celery
+- Redis Queue (RQ)
+
+[Spans]({%- link _documentation/apm/apm-glossary.md -%}#span) are instrumented for the following operations within a [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction):
+
+- Database that uses SQLAlchemy or the Django ORM
+- HTTP requests made with `requests` or the `stdlib`
+- Spawned subprocesses
+
+If you want to enable all relevant transactions automatically, you can use this alternative configuration (currently in alpha):
+
+```python
+import sentry_sdk
+
+sentry_sdk.init("___PUBLIC_DSN___", _experiments={"auto_enabling_integrations": True})
+```
+
+**Manual Instrumentation**
+
+To manually instrument certain regions of your code, you can create a [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction) to capture them. 
+
+The following example creates a transaction for a scope that contains an expensive operation (for example, `process_item`), and sends the result to Sentry:
+
+```python
+import sentry_sdk
+
+while True:
+  item = get_from_queue()
+
+  with sentry_sdk.start_span(op="task", transaction=item.get_transaction()):
+      # process_item may create more spans internally (see next examples)
+      process_item(item)
+```
+
+**Adding More Spans to the Transaction**
+
+The next example contains the implementation of the hypothetical `process_item` function called from the code snippet in the previous section. Our SDK can determine if there is a current open `transaction` and add all newly created [spans]({%- link _documentation/apm/apm-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the `transaction`.
+
+You can choose the value of `op` and `description`.
+
+```python
+import sentry_sdk
+
+def process_item(item):
+    
+  # omitted code...
+  with sentry_sdk.start_span(op="http", description="GET /") as span:
+      response = my_custom_http_library.request("GET", "/")
+      span.set_tag("http.status_code", response.status_code)
+      span.set_data("http.foobarsessionid", get_foobar_sessionid())
+```
+
+### JavaScript
+
+To access our tracing features, you will need to install our APM integration:
+
+```bash
+$ npm install @sentry/browser
+$ npm install @sentry/apm
+```
+
+**Sending Traces**
+
+Tracing resides in the `@sentry/apm` package. You can add it to your `Sentry.init` call:
+
+```javascript
+import * as Sentry from '@sentry/browser';
+import { Integrations as ApmIntegrations } from '@sentry/apm';
```

Comment:
@dashed @markstory technically this says "apm"

Reply:
Should we leave it?

Reply:
yup. all good.

Reply:
Yep please leave it 👍